File: all/abc279/abc279_a/abc279_a.cs
using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    int ans = 0;

    for(int i = 0; i < s.Length; i++)
    {
      if(s[i] == 'v')
      {
        ans++;
      }
      else
      {
        ans += 2;
      }
    }

    Console.WriteLine(ans);
  }
}

File: all/abc279/abc279_a/abc279_a.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    int n = s.length();

    int ans = 0;
    for(int i = 0; i < n; i++) {
      ans += s.charAt(i) - 117;
    }

    System.out.println(ans);
  }
}